internal/runners/activate/activate.go

```diff
@@ -81,8 +60,6 @@ func (r *Activate) run(params *ActivateParams, activatorLoop activationLoopFunc)
 		}
 	}
 
-	go sendProjectIDToAnalytics(params.Namespace, filepath.Join(targetPath, constants.ConfigFileName))
```

Comment:
Should we have another event for activation to replace this one?

Reply:
Martin is adding it in this PR:

https://github.com/ActiveState/cli/pull/857/files